cli/dcoscli/package/main.py

```diff
@@ -401,7 +401,12 @@ def _list(json_, endpoints, app_id, package_name):
 
             results.append(pkg_info)
 
-    emitting.publish_table(emitter, results, tables.package_table, json_)
+    if results:
+        emitting.publish_table(emitter, results, tables.package_table, json_)
+    else:
+        msg = ("There are currently no installed packages. "
+               "Please use `dcos package install` to install a package.")
+        emitter.publish(msg)
```

Comment:
This is going to break scripts. E.g. `dcos package list --json | jq ...`. jq will break because it can't parse the string `There are currently...`.